### ISSUE TITLE
Updater from portable version installs installer version

### ISSUE THREAD

**Issue by klepp0906** [WowUp/WowUp.CF#20]
**Describe the bug**
Running the portable version of the app (which is technically not portable as it saves settings in %appdata% as opposed to alongside the exe but i digress)  it prompts for an update but when you accept, it will install the installer version.

**Expected behavior**
It would be preferable to have the portable executable downloaded/extracted to the same directory that the current one exists.  Or opening to github, or just a notification.  Anything but a button i cant get rid of that force installs the installer version.  as a bonus a way to disable the update checking altogether would be great. 

either way thank you for taking the time to work on wowup, im glad it exists as an alternative. 

EDIT: this time i did not accept and closed it with the intent of coming to download the exe manually.  It appears to have installed it anyhow.  apparently the updater auto downloads the exe and upon closing installs it i assume.

**Comment by Unrepentant-Atheist** [comment]
Yeah, this shit needs to stop. I did not give the app any permissions to auto-update it itself and install itself into c:\user\xxx , that's an absolute shite way to update a software. Unless I tell and allow it so, it shouldn't have any permissions to install/update itself.
